boltdesignsystem/bolt
# DS-590 Notifications component

## Jira

https://pegadigitalit.atlassian.net/browse/DS-590

## Summary

Adds a Notifications component to display user notifications.

## Details

1. Twig template is built out without JS. Notifications JS will be handled on the Drupal side.
2. Changed Switch Button's `on` prop to follow conventions. Boolean props default to `false`.

## How to test

Run the branch locally and check the Notifications docs. Make sure all props work as expected.
